docs/asciidoc/modules/ROOT/pages/overview/apoc.json/apoc.json.path.adoc

```diff
@@ -23,7 +23,8 @@ apoc.json.path(json :: STRING?, path = $ :: STRING?, pathOptions = null :: LIST?
 | Name | Type | Default 
 |json|STRING?|null
 |path|STRING?|$
-|pathOptions|LIST? OF STRING?|[]
+|pathOptions|LIST? OF STRING?|null
+|failOnError|BOOLEAN?|true
 |===
```

Comment:
I'm looking at the other json functions and notice that those seem to take failOnError in a config object. Sometimes the config object only has this parameter, such is the case for apoc.load.jsonParams. 

To me it seems like it would be better to have this option in a Config map for consistency. What do you think?

Reply:
Yeah, it makes sense.
I've now created a `JsonConfig`, similar e.g. to `CompressionConfig`, to handle any future configurations.